evap/staff/tests/test_views.py

```diff
@@ -1953,6 +1976,21 @@ def test_textanswers_full_view(self):
             page = self.app.get(self.url + '?view=full', user=self.manager, status=200)
             self.assertContains(page, self.answer)
 
+    @override_settings(EVALUATION_END_OFFSET_HOURS=26)
```

Comment:
as discussed in this pr, we don't expect this value to be more than 24 hours. so maybe it's a bit strange to have this test value in here?

Reply:
(I think the idea here was to make sure the test even runs in the weirdest of time-zone-change and leap-second combinations. If it makes sense to keep 26 here, we'd probably want to add an explaining comment)

Reply:
okay. i don't think its very important to make the test time change proof (we didn't specify what should happen when reviewing text answers on a time zone change on production 😉), but if you add a comment it should be fine.

Reply:
ok, I hope this is better.

Reply:
probably something like this would explain it even better:

```
# use offset of more than 25 hours to make sure the test doesn't fail even on combined time zone change and leap second
```

Reply:
got it, thanks for the suggestion!

Reply:
thanks!